docs/upgrade-notes/upgrade-rds.md

```diff
@@ -122,6 +122,16 @@ Please read this entire section thoroughly before proceeding to run the second d
 
 :::
 
+:::note
+
+The data migration Lambda described here uses Cumulus' Granule `upsert` logic to write granules from DynamoDB to PostgreSQL. This is particularly notable because granules with a `running` or `queued` status will only migrate a subset of their fields. See the Granule `upsert` logic [here](https://github.com/nasa/cumulus/blob/release-15.0.x/packages/db/src/models/granule.ts#L128).
+
+It is recommended that users ensure their granules are in the correct state before running this data migration. If there are Granules with an incorrect status, it will impact the data migration.
```

Comment:
We should, at least in abstract, cover *how* they might do this.

Reply:
Right, I think a reference to the dead letter archive here is appropriate, and possibly a mention of using the API to update the status if the record is otherwise correct.

Reply:
Added a bit of extra instruction here. It's in a couple commits so here's the most recent full-file: https://github.com/nasa/cumulus/pull/3385/files#diff-4415dfc379dabbb1814e1eed85bf7b84034887b43fa4c197227678aa1b471802

Reply:
Oh interesting. Good catch. We don't want to promise this is a catch-all.

https://github.com/nasa/cumulus/pull/3385/commits/03387d51ea153af866548fea52f0ce6f2acd1d6e